Templates/DefaultProject/Template/AssetBundling/SeedLists/DefaultLevel.seed

```diff
@@ -0,0 +1,22 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "AZStd::vector<SeedInfo, allocator>",
+    "ClassData": [
+        {
+            "assetId": {
+                "guid": "{E989C05E-DB6F-5FDB-92C5-F7BFE3746336}",
+                "subId": 2344825583
+            },
+            "platformFlags": 255,
+            "pathHint": "levels/default/defaultlevel.spawnable"
```

Comment:
Not a big deal, but normally level folder matches the spawnable name.

having the level folder name != the spawnable name will force users to loadlevel in runtime by typing a longer string.
`loadlevel levels/default/defaultlevel.spawnable`

if the folder name and spawnable name match then you can just write
`loadlevel defaultlevel`

Reply:
Yeah, i can change it, lets keep it as close to normal / suggested behavior